include/glow/Backends/TraceEvents.h

```diff
@@ -133,10 +138,34 @@ class TraceContext {
   TraceLevel getTraceLevel() { return traceLevel_; }
   void setTraceLevel(TraceLevel level) { traceLevel_ = level; }
 
-  void logTraceEvent(llvm::StringRef name, llvm::StringRef type = "i",
+  void logTraceEvent(llvm::StringRef name, llvm::StringRef type = "I",
+                     std::map<std::string, std::string> args = {});
```

Comment:
Is `args` supposed to contain extra information in addition to name and type that should be added to the trace? Consider naming this `attributes`, `attrs`, `extraAttrs` or something like that.

Reply:
Yes, it's custom metadata that is displayed in the trace. "args" is the name given in the spec, but agree it could be more descriptive here.